layouts/fastly/helix.vcl

```diff
@@ -1120,7 +1120,7 @@ sub hlx_bereq {
     set bereq.http.Authorization = table.lookup(secrets, "OPENWHISK_AUTH");
     # pass Github Token via X-Github-Token header
     set bereq.http.X-Github-Token = table.lookup(secrets, "GITHUB_TOKEN");
-  } elsif (req.backend == F_GitHub) {
+  } elsif (req.backend == F_GitHub && table.lookup(secrets, "GITHUB_TOKEN")) {
```

Comment:
Why re-reading the table while you have the value in `bereq.http.X-Github-Token` ?

```suggestion
  } elsif (req.backend == F_GitHub && bereq.http.X-Github-Token) {
```

Reply:
I don't think this will work, as `bereq.http.X-Github-Token` is set in a different `elsif` branch, so it would be unset in the condition evaluation.

Reply:
ah yes. In the PR, it is just the previous line :)